src/NEO/ISerializable.h

```diff
@@ -0,0 +1,73 @@
+// Copyright © 2017-2020 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#pragma once
+
+#include "../Data.h"
+#include "../BinaryCoding.h"
+#include "ReadData.h"
+
+namespace TW::NEO {
+
+class ISerializable {
```

Comment:
Please move the static methods into a separate class, to keep the interface a real interface.

Reply:
done